### PR TITLE
The function "exists" has been added in both Linux and Windows wrappers

### PR DESCRIPTION
**Pull request by marciuz**
Hello, I implemented a simple "exists" function. 
The " get($id) " function, if the $id don't exists raise a notice.

### CODE REVIEW

File: src/Linux.php [hunk 1]
```diff
@@ -22,6 +22,17 @@ public function get($id)
...
 
+    /**
+     * Check whether the process identified by ID exists.
+     * @param int
+     * @return bool
+     */
+    public function exists($id)
+    {
+        $out = $this->runCommand("ps u -p ". intval($id));
+        return (is_array($out) && count($out)>0) ? true : false;
```
Comment (tetreum) [inline src/Linux.php]: maybe you can already do:
`return (is_array($out) && count($out) > 0);`
?
(also include spacing between `>` operator --> = ) > 0  :p

File: src/Windows.php [hunk 2]
```diff
@@ -33,6 +33,18 @@ public function get($id)
...
+    /**
+     * Check whether the process identified by ID exists.
+     * @param int
+     * @return bool
+     */
+    public function exists($id)
+    {
+        $processes = $this->runCommand("SELECT * FROM Win32_Process WHERE ProcessId = " . intval($id));
+
+        return (is_array($processes) && count($processes)>0) ? true:false;
```
Comment (tetreum) [inline src/Windows.php]: same here

**Review by tetreum — COMMENTED** [review]
Cool!
I've just made some comments :p

File: src/Windows.php (hunk 2 above)
Comment (marciuz) [reply src/Windows.php]: Fixed

File: src/Linux.php (hunk 1 above)
Comment (marciuz) [reply src/Linux.php]: Fixed

**Pull request merged**